ui/src/dashboards/components/DashboardHeader.tsx

```diff
@@ -67,8 +67,8 @@ export default class DashboardHeader extends Component<Props> {
       zoomedTimeRange: {upper: zoomedUpper, lower: zoomedLower},
       toggleVariablesControlBar,
       isShowingVariablesControlBar,
-      onAddCell,
       onRenameDashboard,
+      onAddCell,
```

Comment:
chasing that line count metric eh?!

Reply:
ha! more like a button was accidentally deleted, then a function was deleted. Then i realized the button should not be deleted and returned everything back XD